src/SFA.DAS.ApprenticePortal.UnitTests/Application/Homepage/Queries/GetApprenticeHomepageQueryTests.cs

```diff
@@ -0,0 +1,69 @@
+﻿using AutoFixture;
+using AutoFixture.NUnit3;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.ApprenticePortal.Application.Homepage.Queries;
+using SFA.DAS.ApprenticePortal.InnerApi.ApprenticeAccounts.Requests;
+using SFA.DAS.ApprenticePortal.InnerApi.ApprenticeCommitments.Requests;
+using SFA.DAS.ApprenticePortal.Models;
+using SFA.DAS.SharedOuterApi.Configuration;
+using SFA.DAS.SharedOuterApi.Interfaces;
+using SFA.DAS.Testing.AutoFixture;
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.ApprenticePortal.UnitTests.Application.ApprenticeAccounts.Queries
+{
+    public class GetApprenticeHomepageQueryTests
+    {
+        private readonly Fixture _fixture = new Fixture();
+
+        [Test, MoqAutoData]
+        public async Task TestGetApprenticeHomepageQuery(            
+            [Frozen] Mock<IApprenticeAccountsApiClient<ApprenticeAccountsApiConfiguration>> accountsApiClient,
+            [Frozen] Mock<IApprenticeCommitmentsApiClient<ApprenticeCommitmentsApiConfiguration>> commitmentsApiClient,
+            GetApprenticeHomepageQueryHandler sut
+            )
+        {
+            //Arrange
+            var apprenticeId = new Guid("80e8b73c-5c3a-11ec-bf63-0242ac130002");
```

Comment:
I would have thought 

... = Guid.NewGuid();

would be easier, but its the same result.

Reply:
changed to NewGuid()